builtins.go

```diff
@@ -90,7 +90,7 @@ func builtinGreater(e *evaluator, xp, yp potentialValue) (value, error) {
 		if err != nil {
 			return nil, err
 		}
-		return makeValueBoolean(left.value > right.value), nil
+		return makeValueBoolean(stringLessThan(right, left)), nil
```

Comment:
Weird that `<` is desugared to not `>=`, but the only string comparison is `<`. I'd imagine for the non-desugared builtin to mirror the string comparison (stringGreaterThan)

Reply:
Good point